README.md

```diff
@@ -1,31 +1,59 @@
-SizeAdjustingTextView
-=====================
 
-This is based on an open source autosizing textview for Android I found a few weeks ago. The initial approach
-didn't resize multiple lines and wasn't maintained to keep up with changes in Android. I decided to go ahead
-and create this as a place to preserve the auto sizing text view as well as giving it a platform for some change
-and to possibly add some features and functionality.
+# SizeAdjustingTextView
 
-## Usage
-To use the SizeAdjustingTextView make sure you have the custom view somewhere in your project.
-From there when using XML Layouts you can simply use the <b>com.some.package.name.SizeAdjustingTextView</b>
-widget and thats it. From there you need to make sure that you set a text size as the baseline for the view. 
-```XML
-<com.example.sizeadjustingtextview.view.SizeAdjustingTextView
-        android:layout_width="fill_parent"
-        android:layout_height="100dp"
-        android:id="@+id/topBox"
-        android:layout_below="@id/text_input"
-        android:gravity="center"
-        android:padding="0dp"
-        android:textSize="60sp" />
+This is based on an open source autosizing text component for Hmos.
+
+## Source
+
+Inspired by [erchenger/SizeAdjustingTextView](https://github.com/erchenger/SizeAdjustingTextView)
+
+## Feature
+This is based on autosizing text component for Hmos.
+
+## Dependency
+1. For using apprate module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
+```groovy
+    dependencies {
+       implementation fileTree(dir: 'libs', include: ['*.jar', '*.har'])
+       implementation project(path: ':sizeadjustingtextview')
+       testImplementation 'junit:junit:4.13'
+       ohosTestImplementation 'com.huawei.ohos.testkit:runner:1.0.0.100'
+}
 ```
-Lastly when using custom views in XML, make sure you have the XLMNS tag in the root of the layout tag. 
-```XML
- xmlns:app="http://schemas.android.com/apk/res/com.example.SizeAdjustingTextView"
+2. For using apprate in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
```

Comment:
Please add APACHE license too for this library as we have added applibgroup copyright for few files..

Reply:
Done.